regulations/static/regulations/css/less/module/comment-confirm.less

```diff
@@ -0,0 +1,42 @@
+
+.comment-confirm-success {
+  .font-regular;
+  padding: 30px 20px;
+  width: 80%;
+
+  .success-alert {
+    background-color: @green_light;
+    display: -webkit-flex;
+    display: flex;
```

Comment:
Over 3% of the total users are running IE below 11, and we don't know how indicative ATF is of our general users. I'm wondering if `flex` is worth it here; it seems like we can accomplish the same end goal without it pretty easily?